### ISSUE TITLE
[Question] useCursor

### ISSUE THREAD

**Issue by mattblackdev** [rijk/meteor-subscribe-suspense#1]
Hey awesome job on this! Where's `useCursor` coming from in the readme examples? Is it the same as doing this: `useTracker(() => MyCollection.find())`?

**Comment by rijk** [comment]
Thanks @mattblackdev  (I know it's lame to comment on this but — **awesome name** 😁)

`useCursor` was from [an upcoming version](https://forums.meteor.com/t/better-hooks-usesubscription-and-usecursor/52989) of react-meteor-data, which I think @captainn ended up abandoning.

**Comment by rijk** [comment]
Yeah, found the original PR: https://github.com/meteor/react-packages/pull/298

> I've broken out the remaining work from this PR to https://github.com/meteor/react-packages/pull/332 (useFind and a new version of useSubscribe added to react-meteor-data) and https://github.com/meteor/react-packages/pull/333 (new package: react-meteor-state with useMeteorState hook).
>
> I'll do the same for react-meteor-accounts and that will obsolete this PR.

That said, I think you should be fine replacing the calls with `useTracker` or something similar — the key for this is replacing the subscription (`useSubscribe` or `Meteor.subscribe`), and then you get the benefit of Suspense meaning the data will be there below that. How you then get the data is probably irrelevant.
